src/components/ErrorPage/ErrorPage.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './ErrorPage.css';
+import { NavLink } from 'react-router-dom';
+
+const ErrorPage = ({ message }) => {
+    return (
+        <section className="error-page">
+            <div className="error-frame">
+                {message === "Server error." ? (
+                    <h2 className="error-message">A server error occured while we were trying to fetch your breweries. Please try again.</h2>
+                ) : (
+
+                    <h2 className="error-message">An unknown error has occured.</h2>)}
+                <NavLink to="/" className="error-link"></NavLink>
+            </div>
+        </section>
+    )
```

Comment:
Button to get back home looks perfect! My only incredibly minor suggestion is that you could be a little more descriptive and maybe say something along the lines of "You've reached a bad URL! Click the button below to go back home" or something like that